pages/dashboard/DashboardHandler.inc.php

```diff
@@ -139,6 +140,34 @@ function index($args, $request) {
 						]
 					]
 				];
+				$lists[$activeListPanel->id]['filters'][] = [
+					'filters' => [
+						[
+							'title' => _('issues'),
+							'param' => 'issueIds',
+							'value' => [],
+							'filterType' => 'pkp-filter-autosuggest',
+							'component' => 'field-select-issues',
+							'autosuggestProps' => [
+									'allErrors' => (object) [],
+									'apiUrl' => $request->getDispatcher()->url($request, ROUTE_API, $context->getPath(), 'issues', null, null, ['roleIds' => [ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR]]),
+									'component' => 'field-select-issues',
+									'description' => '',
+									'deselectLabel' => __('common.removeItem'),
+									'formId' => 'default',
+									'groupId' => 'default',
+									'initialPosition' => 'inline',
+									'isRequired' => false,
+									'label' => __('issues.submissions.issueIds'),
+									'locales' => [],
+									'name' => 'issueIds',
+									'primaryLocale' => 'en_US',
+									'selectedLabel' => __('common.assigned'),
+									'value' => [],
+								]
+							]
+						]
+					];
```

Comment:
We want the issues filter to appear on _every_ submissions list, but only under the following conditions:

1. We're in OJS, not OMP or OPS.
2. The user is an admin, manager, editor or assistant.

In this case, I think the best thing to do is to extend `SubmissionsListPanel.inc.php`, which is only in OJS. I'll leave a comment on that PR with some more details.

Reply:
Resolved in https://github.com/pkp/pkp-lib/issues/6054